api-server/docker-publish.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+tag="${1:=local}"
+container_name="infinite-api-server"
+dockerhub_org="chriswininger"
```

Comment:
Could read the org name from an env variable, and then put it in a Github secret; then if/when we create an Infinite Dockerhub org we just have to change it in one place.

Reply:
I don't think it really needs to be a secret. Nothing secret about it and this keeps the script flexible for local host, at least that was my thinking